lib/highline/import.rb

```diff
@@ -1,3 +1,6 @@
+#!/usr/bin/env ruby
```

Comment:
Dear @mikz I went through the commits and could see that some of the shebangs were just copied and pasted when extracting the classes others were introduced in this commit 0478b5ab1652 where I have put coding information to the files.

Beware to let the shebang on examples and on tests because some tests we can run them by themselves.

I think this has happen before in the history of HighLine ;-)
Removing shebang -> dc2f7cd60d1
Retaining shebangs for examples -> 9f9353815

I think the saying "Those who don't know history are doomed to repeat it" applies to me on this situation :smile: :+1:

As soon as you have the commit, point it to me and I'll cherry pick it. Thank you again.